docs/build-with-powerloom/devnet/health-tracking.md

```diff
@@ -0,0 +1,139 @@
+---
+sidebar_position: 2
+---
+
+# Health Tracking
```

Comment:
Aren't these instructions exactly the same as Snapshotter Node health tracking? If yes, we should either link to those or reorganize the content hierarchy to better suit our needs. 
Don't think it is a good idea to repeat content.